app/controllers/api/memberships_controller.rb

```diff
@@ -9,7 +9,7 @@ def join_group
 
   def invitables
     @memberships = page_collection visible_invitables
-    respond_with_collection scope: { q: params[:q] }
+    respond_with_collection scope: { q: params[:q], include_inviter: false }
```

Comment:
thank you. this looks great.

On Tue, 29 Sep 2015 at 12:11 James Kiesel <notifications@github.com> wrote:

> In app/controllers/api/memberships_controller.rb
> <https://github.com/loomio/loomio/pull/2512#discussion_r40617555>:
>
> > @@ -9,7 +9,7 @@ def join_group
> >
> >    def invitables
> >      @memberships = page_collection visible_invitables
> > -    respond_with_collection scope: { q: params[:q] }
> > +    respond_with_collection scope: { q: params[:q], include_inviter: false }
>
> We had a bug before where inviters were being included in the search
> results, so if I invited you, then my user would come back when searching
> for 'rob'.
>
> This makes it so that the inviter (which, in this context, isn't something
> we really care about), doesn't get returned as a user with the json payload.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2512/files#r40617555>.
>